src/sections/gitops/index.js

```diff
@@ -44,6 +45,38 @@ const GitOpsPage = () => {
         </Row>
       </div>
 
+      <div className="content-container content-container-more-width">
+        <Row className="content-row">
+          <Col lg={6} md={6} className="desc-text-container">
+            <h1 className="heading-1">What is GitOps?</h1>
+            <SquarePoint>
+              GitOps uses Git for defining and tracking infrastructure and app
+              changes.
+            </SquarePoint>
+            <SquarePoint>
+              It automates syncing actual state with Git's desired state.
+            </SquarePoint>
+            <SquarePoint>
+              GitOps ensures consistent, auditable changes using code-defined
+              configurations.
+            </SquarePoint>
+            <div style={{ display: "flex", justifyContent: "center" }}>
+              <Button
+                primary
+                className="learn-more-btn"
+                title="Read More"
+                url="https://layer5.io/resources/cloud-native/what-is-gitops"
+              />
+            </div>
+          </Col>
+          <Col className="hero-image" lg={6} md={6} xs={12} sm={10}>
+            <div className="image-container">
+              <img src={Github_Dark} className="logo" />
```

Comment:
Github_Dark logo only works in dark mode. we need to show different logo in light mode

Reply:
I will get it resolved ☺️